cam2imageGUI.py

```diff
@@ -88,7 +90,7 @@ def MissingInput():
 
 def chooseFiles(fileTypes=None, msg=messages["msgLoadingSVGs"], multiple=True):

```

Comment:
messages["msgLoadingSVGs"] <- never used such things as an argument, you are using a side effect, not a proper argument